controllers/tempo/tempostack_controller.go

```diff
@@ -178,11 +178,29 @@ func (r *TempoStackReconciler) reconcileManifests(ctx context.Context, log logr.
 
 	}
 
+	tenantSecerts := []*manifestutils.GatewayTenantSecret{}
+	if tempo.Spec.Tenants != nil && tempo.Spec.Tenants.Mode == v1alpha1.Static {
+		if err = gateway.ValidateModes(tempo); err != nil {
+			return &status.DegradedError{
+				Message: fmt.Sprintf("Invalid tenants configuration: %s", err),
+				Reason:  v1alpha1.ReasonInvalidTenantsConfiguration,
+				Requeue: false,
+			}
+		}
+
+		tenantSecerts, err = gateway.GetTenantSecrets(ctx, r.Client, req, &tempo)
+		if err != nil {
+			return err
+		}
+		log.Info("gateway secrets set")
```

Comment:
Is this log statement actionable/(needed)?

Reply:
hm.. y, its more a debug log.